### PR TITLE
Update QA and PROD consul hosts

### PR DESCRIPTION
**Pull request by jasonpilz**
### Why is this pull request necessary?
* Consul hosts have been updated.

### How does it fix the issue?
* Update the corresponding .envs

### CODE REVIEW

File: .env.prod [hunk 1]
```diff
@@ -1 +1 @@
-CONSUL_HOST="internal-alb-hfa-us-e-1-p-int-vpc-gen-834019598.us-east-1.elb.amazonaws.com"
+CONSUL_HOST="consul.sesac.com"
```
Comment (MattRooney) [inline .env.prod:1]: Did Eric make these changes in Prod? As of yesterday Ian told me he had not. Also, do you need `https://consul...` here?

**Review by MattRooney — APPROVED** [review]
👍

File: .env.prod (hunk 1 above)
Comment (jasonpilz) [reply .env.prod:1]: hmm good question. @matchbookmac should I hold off on adjusting this in PROD for now?

Comment (jasonpilz) [reply .env.prod:1]: @MattRooney This get interpolated [here](https://github.com/sesac/shipshape/blob/master/bin/provision_by_env#L125). http should re-direct to https.

Comment (matchbookmac) [reply .env.prod:1]: Looks like I was wrong about that. Seems like he's got it going now
https://consul.sesac.com/v1/catalog/service/consul

**Review by matchbookmac — APPROVED** [review]

**Pull request merged**